src/gc/gc.cpp

```diff
@@ -69,6 +69,7 @@ int compact_ratio = 0;
 BOOL reset_mm_p = TRUE;
 
 bool g_fFinalizerRunOnShutDown = false;
+VOLATILE(int32_t) g_fSuspensionPending = 0;
```

Comment:
Nit: This can be in `gccommon.cpp`. If it is here, it is going to be compiled twice (once for server and once for workstation).